custom/icds_reports/static/js/directives/map-or-sector-view/map-or-sector-view.directive.js

```diff
@@ -43,6 +44,9 @@ function MapOrSectorController($scope, $compile, $location, storageService, loca
             }
         });
     }
+    vm.handleMobileDrilldown = function () {
```

Comment:
this is called from here:  https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/static/js/angular-services/baseControllers.service.js#L139